aiidalab_widgets_base/__init__.py

```diff
@@ -4,15 +4,17 @@
 from aiida import load_profile
 load_profile()
 
-from .viewers import viewer  # noqa
 from .codes import CodeDropdown, AiiDACodeSetup, valid_aiidacode_args  # noqa
 from .computers import SshComputerSetup, valid_sshcomputer_args  # noqa
 from .computers import AiidaComputerSetup, valid_aiidacomputer_args  # noqa
 from .computers import ComputerDropdown  # noqa
 from .databases import CodQueryWidget  # noqa
+#from .editors import editor  # noqa
```

Comment:
Is this related to this PR?

Reply:
It is not related, I will add editors in another PR